spec/features/saml/saml_spec.rb

```diff
@@ -167,6 +167,18 @@
   end
 
   context 'visiting /api/saml/logout' do
+    context 'via SP-initiated logout when not logged in to IdP' do
+      it 'does not raise an exception' do
+        request = OneLogin::RubySaml::Logoutrequest.new
+        settings = sp1_saml_settings
+        settings.name_identifier_value = SecureRandom.uuid
+
+        expect do
+          visit request.create(settings)
+        end.to_not raise_error
+      end
+    end
```

Comment:
[This spec](https://github.com/18F/identity-idp/blob/master/spec/features/saml/saml_spec.rb#L263-L277) is not the opposite behavior. This is IdP-initiated logout after the active IdP session has timed out.